examples/complete/main.tf

```diff
@@ -195,7 +197,9 @@ module "cos_bucket2" {
   activity_tracker_crn     = local.at_crn
   create_cos_instance      = false
   existing_cos_instance_id = module.cos_bucket1.cos_instance_id
-  key_protect_key_crn      = module.key_protect_all_inclusive.keys["${local.key_ring_name}.${local.key_name}"].crn
+  # disable retention for test environments - enable for stage/prod
+  retention_enabled   = false
```

Comment:
Remove the line from the comment above the bucket definition that says it has retention as well.